.eslintrc

```diff
@@ -7,12 +7,25 @@
     "prettier/react"
   ],
   "parser": "@typescript-eslint/parser",
-  "plugins": ["jsx-a11y", "fp", "@typescript-eslint", "prettier", "import"],
+  "plugins": [
```

Comment:
Can we revert the formatting changes in this file so that we only see the things that actually changed?